server/src/main/java/org/opensearch/common/settings/KeyStoreWrapper.java

```diff
@@ -134,7 +134,7 @@ private static class Entry {
     private static final String KEYSTORE_FILENAME = "opensearch.keystore";
 
     /** The version of the metadata written before the keystore data. */
-    static final int FORMAT_VERSION = 4;
+    public static final int FORMAT_VERSION = 4;
```

Comment:
Instead of changing the visibility of these members, can we move `KeyStoreWrapper` into the `org.opensearch.cli.keystore` package?

Reply:
Ya initially I have tried that moving `KeyStoreWrapper` to keystore-cli and for server gradle file add a line `  api project(":distribution:tools:keystore-cli")`, but this breaks with.
```
Circular dependency between the following tasks:
:server:compileJava
\--- :distribution:tools:keystore-cli:compileJava
     +--- :server:compileJava (*)
     \--- :distribution:tools:keystore-cli:compileJava (*)

(*) - details omitted (listed previously)
```
Also I assume the KeyStoreWrapper is a wrapper for `org.opensearch.cli.keystore` from `:server ` `org.opensearch.common.settings`.

Reply:
Also I see a mix, example the `SEED_SETTING` is also public `public static final Setting<SecureString> SEED_SETTING = SecureSetting.secureString("keystore.seed", null);`